styles/mixins/_organisms.scss

```diff
@@ -1,19 +1,53 @@
 @use '../variables' as var;
 
+@use 'atoms';
 @use 'layout';
 @use 'molecules';
 
-$-layout-space: var.$layout-space;
+$-color-brand: var.$color-brand;
+$-space: var.$layout-space;
 
 /**
- * An element with a frame, shadow, internal vertical rhythm and padding:
- * serves as a container for card- and panel-like patterns.
+ * A panel is a card that adds an optional header and/or close button positioning.
+ * Panels using an icon-only button as a close button should use the `--closeable`
+ * modifier.
  */
-@mixin card {
-  @include molecules.frame($with-hover: true);
-  @include layout.vertical-rhythm;
+@mixin panel {
+  @include molecules.card;
 
-  padding: $-layout-space;
+  & > header,
+  &__header {
+    @include layout.row($align: center);
+    @include atoms.border(bottom);
+    @include layout.horizontal-rhythm($-space);
```

Comment:
Would an horizontal rhythm be always be suitable if there are several elements in the header?